_examples/table.go

```diff
@@ -0,0 +1,100 @@
+// Copyright 2017 The gocui Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"log"
+
+	"github.com/jroimartin/gocui"
```

Comment:
Can you change this to our repo?

Reply:
Okay.

Reply:
Done ([07aedb](https://github.com/awesome-gocui/gocui/pull/24/commits/07aedbf69057bf3477b4aea4734d7743ad5859ef))